VAMobile/src/screens/BenefitsScreen/Letters/BenefitSummaryServiceVerification/BenefitSummaryServiceVerification.test.tsx

```diff
@@ -232,66 +131,43 @@ context('BenefitSummaryServiceVerification', () => {
     })
   })
 
-  describe('when both the awardEffectiveDate and the monthly payment amount exist', () => {
-    it('should display "Your current monthly award is ${{monthlyAwardAmount}}. The effective date of the last change to your current payment was {{date}}." for that switch', async () => {
-      await waitFor(() => {
-        initializeTestInstance(123, date, 88)
-        expect(testInstance.findAllByType(TextView)[18].props.children).toEqual(
-          'Your current monthly payment is $123.00. The effective date of the last change to your current payment was June 06, 2013.',
-        )
-      })
-    })
-  })
-
   describe('when the monthly award amount does not exist but the awardEffectiveDate does', () => {
-    it('should display "The effective date of the last change to your current award was {{date}}." for that switch', async () => {
-      await waitFor(() => {
-        initializeTestInstance(undefined, date, 88)
-        expect(testInstance.findAllByType(TextView)[18].props.children).toEqual('The effective date of the last change to your current award was June 06, 2013.')
-      })
+    it('should display "Your current monthly award is $0.00. The effective date of the last change to your current award was {{date}}." for that switch', async () => {
+      initializeTestInstance(undefined, date, 88)
+      expect(screen.getByText('Your current monthly payment is $0.00. The effective date of the last change to your current payment was June 06, 2013.')).toBeTruthy()
     })
   })
 
   describe('when the awardEffectiveDate does not exist but the monthly payment amount does', () => {
-    it('should display "Your current monthly award is ${{monthlyAwardAmount}}." for that switch', async () => {
-      await waitFor(() => {
-        initializeTestInstance(123, undefined, 88)
-        expect(testInstance.findAllByType(TextView)[18].props.children).toEqual('Your current monthly payment is $123.00.')
-      })
+    it('should display "Your current monthly award is ${{monthlyAwardAmount}}. The effective date of the last change to your current payment was invalid date." for that switch', async () => {
+      initializeTestInstance(123, undefined, 88)
+      expect(screen.getByText('Your current monthly payment is $123.00. The effective date of the last change to your current payment was invalid date.'))
```

Comment:
```suggestion
      expect(screen.getByText('Your current monthly payment is $123.00. The effective date of the last change to your current payment was an invalid date.'))
```
This should fix the unit test failure

Reply:
fixed